### ISSUE TITLE
Mustache templates are not avaliable, application runs failed

### ISSUE THREAD

**Issue by quietdog** [geddy/geddy#701]
Please follow the simulation steps as below, it is on Win7 x64:

![image](https://cloud.githubusercontent.com/assets/3044063/9000358/a19fa97e-3778-11e5-8dcc-30f19f7fbf6c.png)

![image](https://cloud.githubusercontent.com/assets/3044063/9000354/8d5886c0-3778-11e5-95ff-081c03e337a0.png)

**Comment by quietdog** [comment]
I'm using "-m", then should not require Handlebars Module.

**Comment by phanect** [comment]
Maybe `npm install handlebars` on your project root.

**Comment by phanect** [comment]
BTW is it documented that when you run `geddy -m` you don't have to `npm install handlebars`?
If so, either document or implementation is wrong and we should fix it.

**Comment by phanect** [comment]
@quietdog BTW is it documented that when you run geddy -m you don't have to npm install handlebars?
If so, either document or implementation is wrong and we should fix it.

**Comment by quietdog** [comment]
@phanect Yeah, I was confused when following the document. I just want to use mustache, but why should I include handlebars? Mustache(-m) is different from Handlebars(-H), right?

**Comment by phanect** [comment]
Oh, yes, it looks strange.
